WearTilesKotlin/app/src/main/java/com/example/wear/tiles/messaging/MessagingTileTheme.kt

```diff
@@ -13,22 +13,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.example.wear.tiles
+package com.example.wear.tiles.messaging
 
-import android.app.Application
-import coil.ImageLoader
-import coil.ImageLoaderFactory
-import com.example.wear.tiles.di.AppContainer
+import androidx.wear.tiles.material.Colors
 
-class TilesApplication : Application(), ImageLoaderFactory {
-    internal lateinit var appContainer: AppContainer
-
-    override fun onCreate() {
-        super.onCreate()
-
-        appContainer = AppContainer(this)
-    }
-
-    // Allows Coil ImageLoader singleton to work from anywhere
-    override fun newImageLoader(): ImageLoader = appContainer.imageLoader
+object MessagingTileTheme {
+    private val purple = android.graphics.Color.parseColor("#C58AF9")
```

Comment:
Why parse color over Color(0xFFC58AF9) ? Do we have style guides on this?

Reply:
nah I was going to ask you about this but forgot. Will update. (We don't have style guides on it afaik)

Reply:
Maybe `Color.valueOf(0xFFC58AF9)`.  I'm asking, not that I know our best conventions.  I avoid this by always originating from a MaterialTheme.